### PR TITLE
Move migrations to Contributing

### PR DESCRIPTION
**Pull request by trmartin4**
## Objective

We had Migrations listed under Getting Started.  I was going to direct one of the engineers to the documentation to reference for a PR, and I realized that it probably belongs more under Contributing; this is a part of contributing to the codebase, not getting started.  I left the references to the initial run of `migrate.ps1` as part of Getting Started, but I moved the Migrations section to Contributing.

I also combined both the MSSQL migrations and EF migrations into a single document.

I will update this document when https://github.com/bitwarden/server/pull/2576 is merged to instruct users to use the new migration generator tool.

### CODE REVIEW

File: docs/contributing/database-migrations/index.md [hunk 1]
```diff
@@ -43,4 +49,23 @@ When writing a migration script there are a couple of best practices we follow.
 [repository]:
   https://docs.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design
 [dapper]: https://github.com/DapperLib/Dapper
-[code-style-sql]: ../../../../contributing/code-style/sql.md
+[code-style-sql]: ../code-style/sql.md
+
+## EF Migrations
```
Comment (Hinton) [inline docs/contributing/database-migrations/index.md:54]: EF migrations are documented in https://contributing.bitwarden.com/getting-started/server/database/ef/. We should figure out a good way to organize these.

File: docs/contributing/database-migrations/index.md [hunk 2]
```diff
@@ -43,4 +49,23 @@ When writing a migration script there are a couple of best practices we follow.
 [repository]:
   https://docs.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design
 [dapper]: https://github.com/DapperLib/Dapper
-[code-style-sql]: ../../../../contributing/code-style/sql.md
+[code-style-sql]: ../code-style/sql.md
```
Comment (Hinton) [inline docs/contributing/database-migrations/index.md:52]: These should be last in the file.

**Review by Hinton — COMMENTED** [review]
I wonder if we should move EDD, EF to contributing. Currently the EF docs contains both installing and migration portions.

**Comment by trmartin4** [comment]
> I wonder if we should move EDD, EF to contributing. Currently the EF docs contains both installing and migration portions.

I did move EDD into the Contributing section as well.

I've linked to `Getting Started` from this page to try to avoid repeating ourselves.  Ultimately the same script needs to be run either way, but I'd be happy to change where it lives if you have a preference.

**Review by Hinton — APPROVED** [review]

**Pull request merged**